### ISSUE TITLE
Test compatibility with WordPress 6.3

### ISSUE THREAD

**Issue by bobbingwide** [bobbingwide/oik#217]
In WordPress 6.3 there have been some changes to ensure code conforms to coding standards.
This has affected the internationalization and localization of strings in the shared libraries which are defined with a text domain of null.

There were also a couple of other minor changes to take into account. 
- some changes in WordPress 6.1 that I hadn't previously caught up with
- changes to jQuery version
- changes to the version of genesis-oik

### Proposed method
- Checkout WordPress 6.1, test, and apply any test updates necessary
- Checkout WordPress 6.2, test, and apply any test update necessary
- Checkout each beta and RC version of WordPress 6.3 until the translation tests of strings in shared libraries to bb_BB no longer work.
- Investigate the root cause of the change 
- Either consider suggesting and alternative change OR find a way to implement a new solution to the problem.

### Possible new solution
- Pass pre-translated strings to the functions that need them
- Use constants or globals instead of parameters.

It depends on the number of translatable strings that are implemented in the shared libraries. 
 

See also https://github.com/bobbingwide/bobbingwide/issues/101

**Comment by bobbingwide** [comment]
In WordPress 6.3 there's a change in `get_translations_for_domain()` to set the global `$l10n` array for every domain
even if it nothing is loaded
```
	$l10n[ $domain ] = &$noop_translations;
```

This was affecting the logic in `oik_l10n_gettext()`
The code should be changed to use `is_textdoman_loaded()` as this function now returns `false` when the domain has been loaded as an instance of `NOOP_Translations`.

**Comment by bobbingwide** [comment]
In order to debug the localization problem I needed to ensure the tests worked for WordPress 6.2.2 and then rerun them with WordPress 6.3.  I wrote some simple logic to cater for different versions of jQuery and jQuery-migrate.
The new method `replace_jquery_versions()` is only aware of WordPress 6.2 and 6.3. 
It doesn't bother to check the WordPress version. It just applies the string replacement set the versions to `m.n.p` 
Major - Minor - Patch.

The generated HTML file should now be WordPress version independent.
